package.json

```diff
@@ -2489,6 +2489,316 @@
+												"bindings": [
+													{
+														"key": "e",
+														"name": "Navigate to matching begin/end pair",
+														"type": "command",
+														"command": "latex-workshop.navigate-envpair"
+													},
+													{
+														"key": "s",
+														"name": "Select the current environment name",
+														"type": "command",
+														"command": "latex-workshop.select-envname"
+													},
+													{
+														"key": "S",
+														"name": "Select the current environment content",
+														"type": "command",
+														"command": "latex-workshop.select-envcontent"
+													}
+												]
+											},
+											{
+												"key": "i",
+												"name": "+Insert",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "e",
+														"name": "Close current environment",
+														"type": "command",
+														"command": "latex-workshop.close-env"
+													},
+													{
+														"key": "i",
+														"name": "item",
+														"type": "command",
+														"command": "latex-workshop.shortcut.item"
+													},
+													{
+														"key": "w",
+														"name": "Surround/wrap selection with begin/end",
+														"type": "command",
+														"command": "latex-workshop.wrap-env"
+													}
+												]
+											},
+											{
+												"key": "l",
+												"name": "+Bibtex",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "a",
+														"name": "Align",
+														"type": "command",
+														"command": "latex-workshop.bibalign"
+													},
+													{
+														"key": "s",
+														"name": "Sort",
+														"type": "command",
+														"command": "latex-workshop.bibsort"
+													},
+													{
+														"key": "S",
+														"name": "Sort & Align",
+														"type": "command",
+														"command": "latex-workshop.bibalignsort"
+													}
+												]
+											},
+											{
+												"key": "v",
+												"name": "+View",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "m",
+														"name": "Toggle Match Preview Panel",
+														"type": "command",
+														"command": "latex-workshop.toggleMathPreviewPanel"
+													},
+													{
+														"key": "p",
+														"name": "SyncTeX from cursor",
+														"type": "command",
+														"command": "latex-workshop.synctex"
+													},
+													{
+														"key": "r",
+														"name": "Refresh all viewers",
+														"type": "command",
+														"command": "latex-workshop.refresh-viewer"
+													},
+													{
+														"key": "v",
+														"name": "View Document",
```

Comment:
If this sub menu should be preview, then maybe this should have key `d`?

Reply:
Sure. It would depend on the above.